### PR TITLE
JimWeathersArchives xPath scene/performer scraper

### PR DESCRIPTION
**Pull request by DogmaDragon**
Requested on Discord

### CODE REVIEW

File: scrapers/JimWeathersArchives.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+          - parseDate: 01/02/2006
+      Image:
+        selector: //meta[@name="twitter:image"]/@content
+        postProcess:
+          - replace:
+              - regex: ^(.+)\-.+
+                with: $1-4x.jpg
+      Studio:
+        Name:
+          fixed: BondageCafe
```
Comment (bnkai) [inline scrapers/JimWeathersArchives.yml:33]: Is  the studio actually `BondageCafe` ?

File: scrapers/JimWeathersArchives.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        URL: 
+          selector: //div[@class='gallery_info']/span[@class='update_models']/a/@href         
+  performerScraper:
+    performer:
+      Name:
+        selector: //span[@class='title_bar_hilite']/text()
+      Gender:
+        fixed: female     
+      Image:
+        selector: //img[@id='set-target-13']/@src0_3x
```
Comment (bnkai) [inline scrapers/JimWeathersArchives.yml:49]: the `set-target-xx` is not always 13 (eg `https://www.jimweathersarchives.com/store/models/DianaKnight.html`, `https://www.jimweathersarchives.com/store/models/KendraJames.html`), i suggest switching to something like 
```suggestion
        selector: //img[contains(@class,'model_bio_thumb')]/@src0_3x
```
instead

File: scrapers/JimWeathersArchives.yml (hunk 1 above)
Comment (DogmaDragon) [reply scrapers/JimWeathersArchives.yml:33]: Based on the person who requested the scraper, yes. 
https://discord.com/channels/559159668438728723/651418567475986432/1014170965019729981

File: scrapers/JimWeathersArchives.yml (hunk 2 above)
Comment (DogmaDragon) [reply scrapers/JimWeathersArchives.yml:49]: Noticed that and fixed it on my previous version, but it got lost during some other testing. Thanks for catching it.

**Pull request merged**